census-us/custom-recipes/address-batch-geocoder-us-census-geo/recipe.py

```diff
@@ -0,0 +1,208 @@
+# -*- coding: utf-8 -*-
+import dataiku
+import pandas as pd, numpy as np
+from dataiku import pandasutils as pdu
+import requests 
+#import time
+from dataiku.customrecipe import *
+import sys
+import re
+import geocoder_utils
+import common
+import os
+
+
+logging.info('1/6 Creating base folder...' )
+path_datadir_tmp = dataiku.get_custom_variables()["dip.home"] + '/tmp/'
+P_CENSUS_CONTENT = 'geocoder'
+FOLDER_NAME = 'tmp_census_us_'+ P_CENSUS_CONTENT 
+common.create_folder(path_datadir_tmp,FOLDER_NAME,True)  
+
+input_name = get_input_names_for_role('input')[0]
+
+output_ = get_output_names_for_role('output')[0]
+output_dataset = dataiku.Dataset(output_)
+
+P_COL_STREET = get_recipe_config()['p_col_street']
```

Comment:
why not just retrieve config once and then pass that object to retrieve specific columns- both code hygiene and performance thing